tests/posix/xsi_system_logging/testcase.yaml

```
@@ -0,0 +1,24 @@
+common:
+  filter: not CONFIG_NATIVE_LIBC
+  tags:
+    - xsi.system.logging
+  # 1 tier0 platform per supported architecture
+  platform_key:
+    - arch
+    - simulation
+tests:
+  portability.xsi.system.logging:
+    min_flash: 64
+    min_ram: 32
```

This looks like the same failure signature as the other testsuite, when testing newlib on a platform with only 16 kB of SRAM. 

```suggestion
    - simulation
  min_flash: 64
  min_ram: 32
tests:
  portability.xsi.system.logging:
```